uninstall.php

```diff
@@ -11,4 +11,32 @@
 	exit;
 }
 
-delete_option( 'perflab_modules_settings' );
+if ( is_multisite() ) {
+	$site_ids = get_sites(
+		array(
+			'fields'                 => 'ids',
+			'number'                 => '',
+			'update_site_cache'      => false,
+			'update_site_meta_cache' => false,
+		)
+	);
+
+	foreach ( $site_ids as $site_id ) {
+		switch_to_blog( $site_id );
+		delete_data();
+	}
+	restore_current_blog();
+} else {
+	delete_data();
+}
+
+/**
+ * Delete plugin option data for the current blog.
+ *
+ * @since n.e.x.t
+ */
+function delete_data() {
```

Comment:
As per the plugin function name we should use the `perflab_` prefix in name of the function. Something like `perflab_delete_option` or `perflab_delete_plugin_option`.

cc. @felixarntz

Reply:
In my initial approach, I don't use a helper function for a single line: https://github.com/WordPress/performance/pull/345#issuecomment-1155608644 But I think it was used two times, so Merkys introduced a helper function. If it doesn't make sense, then can we remove it?

Reply:
Thank you for the feedback, removed it

Reply:
Not a blocker, but I also prefer to have a function for this. Yes, it's one line, but IMO it makes sense even to have a function for this to clarify that this needs to be exactly the same logic for a single site as well as for every site in a multisite. Plus it avoids a (admittedly tiny) chance of e.g. a typo in the option name in one of the calls, which in a function call would immediately be flagged as fatal error, but in a string isn't.